### PR TITLE
Rd chart sect one

### PR DESCRIPTION
**Pull request by xjackk**
* https://federal-spending-transparency.atlassian.net/browse/DA-4463
* https://federal-spending-transparency.atlassian.net/browse/DA-5626

I changed the text a bit on the legend on Mobile to have it fit properly. Not a huge change, just abbreviated a bit.

lots of svgs here. (^:

### CODE REVIEW

**Comment by brett-varney** [comment]
legend layout and spacing look off to me; mock:
image

PR:
image

**Comment by brett-varney** [comment]
Chart looks & resizes nice. Show Details is not keyboard-accessible (not in tab order). Also hovering over the border of the details box turns it all opaque (not transparent) blue and the cursor to pointer-hand, but those don't happen when moving directly inside the box.

File: src/page-sections/rd-and-contracting/spending/spendingChart.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  handleWindowSizeChange = () => {
+    this.setState({ bWidth: window.innerWidth });
+  };
+  
+  render() {
+    let bWidth = this.state.bWidth;
+    let isTablet = bWidth <= 768 && bWidth >= 576;
+    let isMobile = bWidth <= 576;
+    let isDesktop = bWidth >= 769;
```
Comment (brett-varney) [inline src/page-sections/rd-and-contracting/spending/spendingChart.jsx:33]: These don't follow the style guide breakpoints:
image

Comment (xjackk) [reply src/page-sections/rd-and-contracting/spending/spendingChart.jsx:33]: How do they not? mobile is 576, tablet is 768? desktop is anything greater than that?

**Pull request closed without merging**